packages/dashboard-frontend/src/services/bootstrap/__tests__/index.spec.tsx

```diff
@@ -104,16 +104,18 @@ describe('Dashboard bootstrap', () => {
     expect(mockGet).toHaveBeenCalledWith('/dashboard/api/namespace/test-che/events', undefined);
     expect(mockGet).toHaveBeenCalledWith('/dashboard/api/namespace/test-che/pods', undefined);
     expect(mockGet).toHaveBeenCalledWith('/dashboard/api/cluster-config');
+    // todo: figure out why ws tests fail on github actions
```

Comment:
just wondering if we started to have this failures on this PR or this happened before ?

Reply:
Yes. We have had this failure before(time to time) and I can not reproduce this error locally.

Reply:
This test fails sometimes when running on GH, and usually on the second run it passes.